relays/substrate/src/cli.rs

```diff
@@ -38,6 +39,52 @@ pub enum Command {
 		#[structopt(flatten)]
 		prometheus_params: PrometheusParams,
 	},
+	/// Submit message to given Rialto -> Millau lane.
+	SubmitMillauToRialtoMessage {
+		#[structopt(flatten)]
+		millau: MillauConnectionParams,
+		#[structopt(flatten)]
+		millau_sign: MillauSigningParams,
```

Comment:
Is there any way to give example defaults here?

Reply:
I remember Tomek has had some problems with default values when he worked on eth2sub/sub2eth deployments, which he has mentioned in #162. So I haven't added any default values (except for prometheus cfg).

Having default values is imo better than not having them. But if it causes problems, then let's add all arguments explicitly.